src/mixins/_mixins.scss

```diff
@@ -342,3 +342,23 @@
   color: $color;
   font-size: $fd-size;
 }
+
+@mixin fd-icon-overwrite {
```

Comment:
Not sure about naming. I think this is a similar use case to our other mixins and `fd-icon-overwrite` does not overwrite anything by itself, so this should rather be just `fd-icon`.

Reply:
we added `reset` to icons so now we need to overwrite properties like `border:0`, `margin:0`, etc. `

Reply:
`fd-icon` is already used